File: MusicAssignment/MusicAssignment/ViewController.m
//
//  ViewController.m
//  MusicAssignment
//
//  Created by mac mini on 10/9/14.
//  Copyright (c) 2014 ParadigmCreatives. All rights reserved.
//

#import "ViewController.h"
#import "AppDelegate.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    [[UIApplication sharedApplication] beginReceivingRemoteControlEvents];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction)pauseAction:(id)sender {
    AppDelegate *appDelegate = [[UIApplication sharedApplication] delegate];
    [appDelegate.player pause];
}

- (IBAction)playAction:(id)sender {
    AppDelegate *appdelegate = [[UIApplication sharedApplication] delegate];
    appdelegate.player.delegate=self;
      [appdelegate.player1 stop];
       [appdelegate.player play];
    
}

- (IBAction)pause1:(id)sender {
    
    AppDelegate *appdel=[[UIApplication sharedApplication]delegate];
    [appdel.player1 pause];
}

- (IBAction)play1:(id)sender {
    
    AppDelegate *appdel=[[UIApplication sharedApplication]delegate];
    appdel.player1.delegate=self;
       [appdel.player stop];
    [appdel.player1 play];
}
@end

File: Music-Swift/Music-Swift/ViewController.swift
//
//  ViewController.swift
//  Music-Swift
//
//  Created by mac mini on 10/31/14.
//  Copyright (c) 2014 ParadigmCreatives. All rights reserved.
//

import UIKit
import AVFoundation
import AVKit


class ViewController: UIViewController , AVAudioPlayerDelegate {
    var player = AVAudioPlayer()
    override func viewDidLoad() {
        super.viewDidLoad()
                
       
        var url = NSURL(fileURLWithPath: NSBundle.mainBundle().pathForResource("Song", ofType: "mp3")!)
        var error:NSError?
        player = AVAudioPlayer(contentsOfURL: url, error: &error)
        player.prepareToPlay()
        player.delegate=self
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    @IBAction func playClicked(sender: AnyObject) {
        player.play()
    }

    @IBAction func stopClicked(sender: AnyObject) {
        player.stop()
    }
    }

